db.sql

```diff
@@ -1505,6 +1606,20 @@ CREATE TABLE `ticket_attachments` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `ticket_products`
```

Comment:
Ticket products seems to be a new table defined in db.sql, but seems to not be added during the database upgrade?

Reply:
Good catch. This was resolved.